tests/interfaces/test_jax_qnode.py

```diff
@@ -856,17 +888,17 @@ def circuit():
         assert isinstance(res[1], jax.Array)
         assert res[1].shape == (10,)
 
-    def test_counts(self, dev_name, diff_method, mode):
+    def test_counts(self, dev_name, diff_method, grad_on_execution):
         """Test counts works as expected"""
-        if mode == "forward":
-            pytest.skip("Sampling not possible with forward mode differentiation.")
+        if grad_on_execution:
```

Comment:
Same as above

Reply:
I updated it in the new tests wiht `is True`